modules/UI/shared_video/SharedVideo.js

```diff
@@ -246,28 +251,24 @@ export default class SharedVideoManager {
      * Process attributes, whether player needs to be paused or seek.
      * @param player the player to operate over
      * @param attributes the attributes with the player state we want
-     * @param playerPaused current saved state for the player
      */
-    processAttributes (player, attributes, playerPaused)
+    processVideoUpdate (player, attributes)
```

Comment:
Isn't it better to use either "this.player" either the "player" parameter instead of both? I personally prefer "this.player". WDYT?